src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
enum name should be in singular
```suggestion
public enum Color {
```